x-pack/packages/observability/alerting_test_data/src/constants.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const DATA_VIEW_ID = 'data-view-id';
+export const FIRED_ACTIONS_ID = 'custom_threshold.fired';
+export const ALERT_ACTION_INDEX = 'test-alert-action-index';
+
+export const KIBANA_DEFAULT_URL = 'http://127.0.0.1:5601/kibana';
```

Comment:
@maryam-saeidi This url didn't work for me. I used `'http://127.0.0.1:5601/'` instead to be able to run the create_observability_rules.js script. I will leave another comment in the PR with the error I got. 

Is this a correct `KIBANA_DEFAULT_URL`? How did it work for you?

Reply:
Good catch, this works for me because I hard codded the path to `kibana`, will fix it 👍🏻